src/type-translator.ts

```diff
@@ -442,7 +442,7 @@ export class TypeTranslator {
           let member = type.symbol.members.get(field);
```

Comment:
Better: make this `.get(field)!`.
We know member is never-null because we are iterating the keys of the map.